packages/admin-graphql-api-utilities/src/index.ts

```diff
@@ -1,4 +1,21 @@
-const GID_REGEXP = /\/(\w+(-\w+)*)(?:\?(?:[\w-_]+=[\w-_]+&)*(?:[\w-_]+=[\w-_]+))?$/;
+const GID_TYPE_REGEXP = /\/(\w+)\/(\w[\w-]*)(\?([\w-]+=[\w-]+(&[\w-]+=[\w-]+)*))*$/;
```

Comment:
Do we need as much validation for an ID? Can we simplify the regex with something like : ```gid:\/\/[\w]+\/(\w*)\/.*```